R/ResampleInstances.R

```diff
@@ -67,3 +67,78 @@ instantiateResampleInstance.SpRepCVDesc = function(desc, size, task = NULL, coor
   g = as.factor(rep(seq_len(desc$reps), each = folds))
   makeResampleInstanceInternal(desc, size, train.inds = train.inds, test.inds = test.inds, group = g)
 }
+
+
+instantiateResampleInstance.FixedCVDesc = function(desc, size, task = NULL, coords) {
+  initial.window.abs = floor(desc$initial.window * size)
+  assertInt(initial.window.abs, lower = 1)
+  if (size - initial.window.abs < desc$horizon)
+    stop(catf("The initial window is %i observations while the data is %i observations. \n
+              There is not enough data left (%i observations) to create a test set for a %i size horizon",
+              initial.window.abs, size, initial.window.abs - size, desc$horizon))
+  skip = desc$skip
+  stops  = (seq(size))[initial.window.abs:(size - desc$horizon)]
+  starts = stops - initial.window.abs + 1
+  train.inds = mapply(seq, starts, stops, SIMPLIFY = FALSE)
+  test.inds  = mapply(seq, stops + 1, stops + desc$horizon, SIMPLIFY = FALSE)
+
+  thin = function(x, skip = 0) {
```

Comment:
Is the default for `skip` needed here? Looks like it will never be called with 0.

Reply:
Nope, removed